build/Linux+BSD/portable/copy-libs

```diff
@@ -93,6 +93,8 @@ getCrossPlatformDependencies() {
 
   copyQt plugins/sqldrivers/libqsqlite.so
 
+  copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so
```

Comment:
OK!  (I mistakenly assumed was also windows when I saw this suspiciously similar error on Windows https://musescore.org/en/node/59856)